governance/02-issues-workflow.md

```diff
@@ -50,7 +52,7 @@ There are eight different stages of the triage:
 | Security | Specify the [CVSSv3](https://www.first.org/cvss/calculator/3.0) severity with the support of the security team. | security/{SEVERITY} |
 | SIG/WG | Clarify which SIG or WG is involved in this issue and is responsible for the further follow-up on the issue. | sig/{SIG_NAME}, wg/{WG_NAME}|
 | Priority | Prioritize issues in the general Kyma backlog to select those which are the most critical and should be taken as first. | priority/critical |
-| Area | Clarify which capabilities are involved in a given issue. | area/{CAPABILITY_NAME} |
+| Area | Clarify which capabilities or qualities are involved in a given issue. | area/{CAPABILITY_NAME} or area/quality |
```

Comment:
```suggestion
| Area | Clarify which capabilities are involved in a given issue. | area/{CAPABILITY_NAME} |
|Quality | Clarify which qualities are involved in a given issue. | quality/{QUALITY_NAME} |
```
Now the quality labels are separate from the area ones. Will it be changed?

Reply:
quality labels will be removed. Instead we want to have one label for all qualities which is area/quality. And one label for a performance which will be area/performance